app/assets/javascripts/views/vacation_request.js

```diff
@@ -0,0 +1,31 @@
+App.Views.VacationRequest = Backbone.View.extend({
+  tagName: 'tr',
+  template: JST['templates/vacation_request'],
+
+  initialize: function() {
+    var start     = new Date(this.model.get('start'));
+    var duration  = this.model.get('duration');
+    var finish    = this.model.get('end');
+
+    d1 = Date.UTC(start.getFullYear(), start.getMonth(), start.getDate());
+    d2 = d1 + duration * 24 * 3600 * 1000;
+
+    if (finish === null) {
+      finish = new Date(d2);
+      finish = finish.toJSON().slice(0,10);
```

Comment:
Well, if you don't, you'll won't get dates in dedicated input of type date.
For details, see http://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_date